src/gui/qgstemporalcontrollerwidget.cpp

```diff
@@ -225,17 +253,26 @@ void QgsTemporalControllerWidget::togglePlayForward()
 {
   mPlayingForward = true;
 
-  if ( mNavigationObject->animationState() != QgsTemporalNavigationObject::Forward )
+  if ( mNavigationObject->animationState() != Qgis::AnimationState::Forward )
   {
     mStopButton->setChecked( false );
     mBackButton->setChecked( false );
     mForwardButton->setChecked( true );
+
+    mStopButton_2->setChecked( false );
+    mBackButton_2->setChecked( false );
+    mForwardButton_2->setChecked( true );
```

Comment:
I agree it's gross. But I don't particularly like the hiding approach either -- when we do that we end up with slightly misspaced layouts, because qt still accounts for the inter-widget spacing even for hidden widgets. So we end up with double / triple / ... spacing instead of single. It's ok in some contexts, but looks bad here :P 

And ultimately we end up with just as complex logic about hiding /showing / changing behaviour of existing widgets. :man_shrugging:

Reply:
@nirvn -- ok, i've cleaned this up, and moved the button strip out to a reusable separate widget.

Reply:
@nyalldawson , nice!